src/Core/Framework/Adapter/Cache/CacheInvalidationSubscriber.php

```diff
@@ -502,37 +501,6 @@ public function invalidatePropertyFilters(EntityWrittenContainerEvent $event): v
         $this->cacheInvalidator->invalidate([...$this->getChangedPropertyFilterTags($event), ...$this->getDeletedPropertyFilterTags($event)]);
     }
 
-    /**
-     * @deprecated tag:v6.7.0 - reason:remove-subscriber - Will be removed, use invalidateProduct instead
-     */
-    public function invalidateReviewRoute(ProductChangedEventInterface $event): void
```

Comment:
please also do not touch this. it is part of the cache_rework which is currently done by me